### ISSUE TITLE
nsb format bug when run with table_test

### ISSUE THREAD

**Issue by erickaiwu** [rishi-kulkarni/nsb-toolbox#46]
nsb format with no additional arguments returns error message : "ValueError: tuple.index(x): x not in tuple", works fine when --capitalize is used however.

![image](https://github.com/rishi-kulkarni/nsb-toolbox/assets/168475240/58f62453-4b3f-4775-849a-4cb8ec537f8d)
![image](https://github.com/rishi-kulkarni/nsb-toolbox/assets/168475240/e970fffd-53f7-4178-908c-9b4bb0eb0aeb)

**Comment by rishi-kulkarni** [comment]
Thanks, good catch. It seems that when I stopped auto-capitalization of the answer line (to help handle chemical formulae and math equations), it made it so the W, X, Y, or Z in the answer line MUST be capitalized. This should be fixable by changing this line:

https://github.com/rishi-kulkarni/nsb-toolbox/blob/cdbdad467006d86c80465d72ecd1ce86c835ed85/nsb_toolbox/tables.py#L553

to:

```python
choice_num = CHOICES.index(test_choice_match.group(1).upper() + ")")
```

I'm happy to review and merge a PR if you open one, otherwise I can probably push a fix this weekend. It'd probably be good to add a test that fails on current master due to this behavior and make sure it passes on the PR branch.

**Comment by rishi-kulkarni** [comment]
This has been fixed and release in v0.5.5. Thanks for the report!
